EIPS/eip-3664.md

```diff
@@ -0,0 +1,383 @@
+---
+eip: 3664
+title: Next generation metaverse NFT
+description: This is a standard for next generation metaverse NFTs, which extends ERC-1155.
+author: TylerZ (@tylerztl), TY (@nimocat)
+discussions-to: https://ethereum-magicians.org/t/eip-3664-next-generation-game-nft-standard/6738
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-07-17
+requires: 20, 1155
+---
+
+## Abstract
+In this proposal, we created the next gen component-NFT standard, which makes up for the problem that [ERC-721](./eip-721.md) and [ERC-1155](./eip-1155.md) can only support unchangeable attributes of NFTs. component-NFT can make the attributes in a NFT have five characteristics: transferable, upgradeable, changeable, combinable and evolvable. By using this standard NFTs in any fields, users can create any valuable dapps based on component-NFT standard.
```

Comment:
The abstract should contain a _technical_ summary of the EIP, so readers can get a quick idea of what changes the EIP proposes. Should probably mention something about "attributes" becoming their own entity, that are attached to NFTs?